var/spack/repos/builtin/packages/magma/package.py

```diff
@@ -78,7 +78,7 @@ class Magma(CMakePackage, CudaPackage, ROCmPackage):
     patch("magma-2.5.0.patch", when="@2.5.0")
     patch("magma-2.5.0-cmake.patch", when="@2.5.0")
     patch("cmake-W.patch", when="@2.5.0:%nvhpc")
-    patch("0001-fix-magma-build-error-with-rocm-6.0.0.patch", when="@2.7.2 ^hip@6.0.0 + rocm")
+    patch("0001-fix-magma-build-error-with-rocm-6.0.0.patch", when="@2.7.2 ^hip@6.0 + rocm")
```

Comment:
(follow-up) I'm guessing this is because it also applies for `hip@6.0.2` - that works fine although the patch name is no longer accurate